### ISSUE TITLE
Are we able to skip the cutscenes?

### ISSUE THREAD

**Issue by sarimmehdi** [RobertBeckebans/RBDOOM-3-BFG#446]
The story of this game is really boring and I only want to shoot demons. Sitting through the boring and bland cutscenes ruins all the fun. Are we able to skip cutscenes through this mod?

**Comment by Calinou** [comment]
I think you can skip cutscenes using <kbd>Escape</kbd>. This doesn't seem to apply to the first cutscene with the video though.

**Comment by RobertBeckebans** [comment]
All cutscenes should be skipable.

**Comment by BielBdeLuna** [comment]
I ported the skipping function from dhewm3

**Comment by SEAGitUser** [comment]
There still a problem. Sounds of cinematics keeps playing after skipping.
There is a function idSoundWorld::Skip( time ) for this purposes according to description but what time value to use and when to call it ? Can some one to resolve this?

**Comment by BielBdeLuna** [comment]
I honestly don't remember where the skip function is or how did it work as I ported the code a long time ago, maybe it returns the time it skips?

**Comment by SRSaunders** [comment]
Yes, `idSoundWorld::Skip( time )` is the right function, and I'm pretty sure I fixed this already back in Feb 2022.

See merged commit: https://github.com/RobertBeckebans/RBDOOM-3-BFG/commit/3be85d9c4b539a609c75e9f8b8af31cb237488f4.  The relevant code inside `idGameLocal::SkipCinematic()` is:

```
soundSystem->SetMute( true );
if( !skipCinematic )
{
	skipCinematic = true;
	cinematicMaxSkipTime = gameLocal.time + SEC2MS( g_cinematicMaxSkipTime.GetFloat() );
	// SRS - Skip the remainder of the currently playing cinematic sound
	soundSystem->GetPlayingSoundWorld()->Skip( cinematicMaxSkipTime );
}
```
